python/pylintrc

```diff
@@ -183,10 +183,10 @@ name-group=
 include-naming-hint=no
 
 # Regular expression matching correct function names
-function-rgx=[a-z_][a-z0-9_]{2,30}$
+function-rgx=[a-z_][a-z0-9_]{2,120}$
```

Comment:
Some of the test methods easily exceed 30 characters, particularly if you name them verbosely. I don't see a good reason why test methods should have long names, so increasing this maximum to the max line width.